webapp/src/components/properties/link/link.tsx

```diff
@@ -37,6 +38,7 @@ const URLProperty = (props: Props): JSX.Element => {
                 className='octo-propertyvalue'
                 placeholderText='Empty'
```

Comment:
If the URLProperty is readonly, the placeholder should be blank. If you like, go ahead and make that change with this PR, or we can create a separate issue to track it.

Reply:
@chenilim this was already changed in #1008